charts/influxdb/README.md

```diff
@@ -194,7 +216,7 @@ It is up to the end user to configure their own one-off restore jobs. Below is j
 apiVersion: batch/v1
 kind: Job
 metadata:
-  generateName: influxdb-restore-
+  generateName: influxdb-restore
```

Comment:
Having a `-` between the prefix and the generated id is good, why remove it?

Reply:
Looked like a typo :D